src/obb/events/lobby_events.clj

```diff
@@ -0,0 +1,49 @@
+(ns ^{:added "0.1.0" :author "Pedro Pereira Santos"}
+  obb.events.lobby-events
+  "Manages subscribers and processes lobby events"
+  (:require [clojure.core.async :as async]
+            [com.stuartsierra.component :as component]
+            [obb.models.game-info :as game-info]))
+
+(defprotocol LobbyEvents
+  (add-game [this game-info])
+  (remove-game [this game-info])
+  (subscribe [this subscriber-ch]))
+
+(defrecord LobbyEventSystem [master-ch multiplier]
+
+  component/Lifecycle
+
+  (start [component]
+    (let [master-ch (async/chan)
+          multiplier (async/mult master-ch)]
+      (assoc component :master-ch master-ch
+                       :multiplier multiplier)))
+
+  (stop [component]
+    (async/close! master-ch)
+    (-> component
+        (dissoc :master-ch)
+        (dissoc :multiplier)))
+
+  LobbyEvents
+
+  (add-game [this game-info]
+    (async/go
+      (async/>! master-ch
+                {:action :added
```

Comment:
Nice one, I'll change it.
I added the action because in my mind consumers need to know that. For example the lobby page gets a new event with a game, if its add, adds it to the list, if it's remove, removes it from the list.